src/views/Game/Game.styl

```diff
@@ -38,6 +38,9 @@
     .submit-button {
         white-space: nowrap;
     }
+    .autofocus-submit-button:focus::after {
+        content: ' ⏎';
+    }
```

Comment:
I think if you add this after the above rule:

```
    .autofocus-submit-button:active::after {
        content: none;
    }
```

it'll fix what you're seeing with clicking the submit button when it doesn't already have focus (as it will have focus but it'll also be active during the click, and the active wins since the rule is later).